core/src/components/item-divider/item-divider.scss

```
@@ -67,12 +74,19 @@
 
 .item-divider-inner {
   @include margin(0);
-  @include padding(
-    var(--inner-padding-top),
-    calc(var(--ion-safe-area-right, 0px) + var(--inner-padding-end)),
-    var(--inner-padding-bottom),
-    var(--inner-padding-start)
-  );
+  @include padding(var(--inner-padding-top), null, var(--inner-padding-bottom));
```

Nit: Can we add `null` at the end? I know it doesn't make any functional difference, but it threw me off only seeing 3 values here.
```suggestion
  @include padding(var(--inner-padding-top), null, var(--inner-padding-bottom), null);
```